src/bridging/BridgingSdk/getQuoteWithBridge.ts

```diff
@@ -195,15 +197,19 @@ export async function getQuoteWithBridge<T extends BridgeQuoteResult>(
     }
   }
 
-  // Sign the hooks with bridgeHookSigner if provided
-  const result = await signHooksAndSetSwapResult(bridgeHookSigner ? getSigner(bridgeHookSigner) : signer)
+  const result = await signHooksAndSetSwapResult(
+    // Sign the hooks with bridgeHookSigner if provided
+    bridgeHookSigner ? getSigner(bridgeHookSigner) : signer,
+    // Use default gas limit if bridgeHookSigner is provided
+    bridgeHookSigner ? BigInt(gasLimit) : undefined
```

Comment:
Did you mean to use `defaultGasLimit` here instead?

Reply:
Thanks, added more info and renamed const:
https://github.com/cowprotocol/cow-sdk/pull/312/commits/6320bffcaac623814cb08746463d6feb5b915bae